app/controllers/api_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ApiController < ApplicationController
+  protect_from_forgery with: :null_session
```

Comment:
Since auth is JWT based and there is currently no user auth for the API, I wonder if CSRF and related methods affect us at all?

Reply:
I think that will this in place CSRF should not apply to these endpoints, which is what I was going for. Without an actual client, I don't know for sure - happy to address this once engineering hits this part of the process.